cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForShuntCompensators.java

```diff
@@ -75,7 +77,8 @@ private void apply(ShuntCompensator shuntCompensator, CgmesRegulatingControlForS
         // But no regulating control information has been found
         // We create default regulation data
         if (rcId == null) {
-            context.missing("Regulating Control ID not defined");
+            // No need to report as missing, just log a message
```

Comment:
Can you remove both comment "// No need to report as missing, just log a message". If you are sure about the solution, you don't need a comment.